### PR TITLE
Line edits chapter 0 and chapter 1

### PR DESCRIPTION
**Pull request by TeganAli**

### CODE REVIEW

**Review by mcasperson — APPROVED** [review]
Looks good to me. Thanks Tegan!

File: _includes/chapter1.html [hunk 1]
```diff
@@ -265,15 +297,15 @@ <h3>Octopus workers and worker pools</h3>
 	Up until this point we have been working with the Octopus server. You can think of the server as an orchestrator that ensures deployments are performed in the required environments to the appropriate targets with all the necessary settings and packages.
 </p>
 <p>
-	When performing deployments to on-premises infrastructure, such as a web server hosted on a VM, the actual execution of the deployment is performed through an agent called a tentacle. In keeping with the nautical themes you'll often encounter with Octopus terminology, an Octopus server has many tentacles that reach out into your infrastructure. Where multiple VMs had to be updated with each deployment, multiple tentacles were deployed with a shared role like <b>Web Server</b>. So a deployment configured to update all targets of the role <b>Web Server</b> would reach out across all the tentacles with that role and perform the required changes:
```
Comment (TeganAli) [inline _includes/chapter1.html:268]: @wordlee have read the style guide but unsure whether Tentacle should always be capitalized or not?

File: _includes/chapter0.html [hunk 2]
```diff
@@ -12,15 +12,21 @@ <h1><a id="intro">Welcome</a></h1>
 	With that in mind, pillar number zero, and the prerequisite for all other pillars in this list, is do what works for you.
 </p>
 <p>	
-	We hope you enjoy this discussion, and look forward to shaping the future of pragmatic deployments with your feedback.
+	We hope you enjoy this discussion, and look forward to shaping the future of pragmatic Kubernetes deployments with your feedback.
 </p>
 <h2>Viewing the results of this book</h2>
 <p>
-	An hosted Octopus instance with guest access has been created at <a href="https://tenpillars.octopus.app/app#/Spaces-2">https://tenpillars.octopus.app/app#/Spaces-2</a>. You can browse this instance to see the projects, runbooks, and other configuration discussed in this book.
+	A hosted Octopus instance with guest access has been created at <a href="https://tenpillars.octopus.app/app#/Spaces-2">https://tenpillars.octopus.app/app#/Spaces-2</a>. You can browse this instance to see the projects, runbooks, and other configurations discussed in this book.
```
Comment (TeganAli) [inline _includes/chapter0.html:19]: @wordlee unsure if Runbooks should be capitalized or not?

**Comment by mcasperson** [comment]
Apologies Tegan - I merged this to get the changes for an initial print of the book. If there are further changes feel free to reopen the PR.

**Pull request merged**